pipeline/src/node_properties/mod.rs

```diff
@@ -45,20 +45,14 @@ where
     }
 }
 
-fn serialize_required<T, S>(field: &T, serializer: S) -> Result<S::Ok, S::Error>
-where
-    S: serde::ser::Serializer,
-    T: Display,
-{
-    field.to_string().serialize(serializer)
-}
-
 pub mod encode_properties;
 pub use encode_properties::EncodeProperties;
 pub mod stream_rtsp_out_properties;
 pub use stream_rtsp_out_properties::StreamRtspOutProperties;
+pub use stream_rtsp_out_properties::StreamRtspOutRuntime;
```

Comment:
Maybe we should merge these? Although that's a refactoring indepent of the changes made so far.

```suggestion
pub mod stream_rtsp_out_properties::{self, StreamRtspOutProperties, StreamRtspOutRuntime};
```

Reply:
Yeah, can do in a separate commit